tests/format/less/less/less.less

```diff
@@ -1694,3 +1694,20 @@ label {
 // #8177
 .hello {
   &:extend(.input:disabled)}
+
+// no space after unary minus, #13998
+.unary_minus_single {
+  margin: -(@a);
+}
+
+.unary_minus_multi_1 {
+  margin: 0 -(@a);
+}
+
+.unary_minus_multi_2 {
+  margin: 0  -( @a + @b );
+}
+
+.binary_minus {
+  margin: 0 - (@a);
```

Comment:
try https://lesscss.org/less-preview/ less online compiler, a space is changing the meaning of the data